interfaces/seccomp/backend.go

```diff
@@ -81,6 +83,19 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	if err != nil {
 		return fmt.Errorf("cannot synchronize security files for snap %q: %s", snapName, err)
 	}
+
+	for baseName := range content {
+		in := filepath.Join(dirs.SnapSeccompDir, baseName)
+		out := filepath.Join(dirs.SnapSeccompDir, baseName[:len(baseName)-3]+".bpf")
```

Comment:
I don't have a strong opinion except to say that I think it should be consistent with what we have. That suggests to me one of the following (not in order of preference):
- /var/cache/snapd/seccomp (similar to /var/cache/apparmor for snappy apparmor cache files)
- /var/lib/snapd/seccomp/cache (ie, put it alongside /var/lib/snapd/seccomp/profiles; if you prefer /var/lib/snapd/seccomp/bpf instead, that's fine too)